lib/team_city_formatter/logger.rb

```diff
@@ -13,7 +13,7 @@ def test_suite_finished(test_suite_name)
     end
 
     def test_started(test_name)
-      render_output("##teamcity[testStarted name='#{teamcity_escape(test_name)}' timestamp='#{timestamp}' captureStandardOutput='true']")
+      render_output("##teamcity[testStarted name='#{teamcity_escape(test_name)}' captureStandardOutput='true' timestamp='#{timestamp}']")
```

Comment:
@kevinrood I just saw in the documentation and everywhere the timestamp was in the end. It seems more consistent for me but I can revert this line if you want.